web/src/lib/i18n/en.json

```diff
@@ -880,6 +880,7 @@
   "reassing_hint": "Assign selected assets to an existing person",
   "recent": "Recent",
   "recent_searches": "Recent searches",
+  "redirecting": "Redirecting...",
```

Comment:
This seems unused

Reply:
Oh yeah you are right. I had it at one point.